<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="Titlebar" name="titlebar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="center" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>0</number>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>0</number>
          </property>
          <widget class="NDirTreeview" name="dirTreeView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollBar:vertical
 {
         width:6px;
         background:rgba(0,0,0,0%);
          margin:0px,0px,0px,0px;
          padding-top:0px;
          padding-bottom:0px;
 }
QScrollBar::handle:vertical
{
          width:6px;
          background:rgba(0,0,0,25%);
          border-radius:4px;
          min-height:20;
}
QScrollBar::handle:vertical:hover
{
          width:6px;
          background:rgba(0,0,0,50%);
          min-height:20;
}
QScrollBar::add-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url(&quot;&quot;);
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
          background:rgba(0,0,0,10%);
}
                                                       </string>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
          <widget class="NoteTreeView" name="fileTreeView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QScrollBar:vertical
 {
         width:6px;
         background:rgba(0,0,0,0%);
          margin:0px,0px,0px,0px;
          padding-top:0px;
          padding-bottom:0px;
 }
QScrollBar::handle:vertical
{
          width:6px;
          background:rgba(0,0,0,25%);
          border-radius:4px;
          min-height:20;
}
QScrollBar::handle:vertical:hover
{
          width:6px;
          background:rgba(0,0,0,50%);
          min-height:20;
}
QScrollBar::add-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url(&quot;&quot;);
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
          background:rgba(0,0,0,10%);
}
                                                       </string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditKeyPressed</set>
           </property>
          </widget>
         </widget>
         <widget class="MarkdownEditor" name="editorView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
         <widget class="QTextBrowser" name="tocView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">a:link{
		text-decoration: none;
}

QScrollBar:vertical
 {
         width:6px;
         background:rgba(0,0,0,0%);
          margin:0px,0px,0px,0px;
          padding-top:0px;
          padding-bottom:0px;
 }
QScrollBar::handle:vertical
{
          width:6px;
          background:rgba(0,0,0,25%);
          border-radius:4px;
          min-height:20;
}
QScrollBar::handle:vertical:hover
{
          width:6px;
          background:rgba(0,0,0,50%);
          min-height:20;
}
QScrollBar::add-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url(&quot;&quot;);
          subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
          height:0px;width:6px;
          border-image:url();
          subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
          background:rgba(0,0,0,10%);
}
                                                       </string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="tabStopWidth">
           <number>10</number>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Titlebar</class>
   <extends>QWidget</extends>
   <header location="global">titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NDirTreeview</class>
   <extends>QTreeView</extends>
   <header location="global">ndirtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>NoteTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">notetreeview.h</header>
  </customwidget>
  <customwidget>
   <class>MarkdownEditor</class>
   <extends>QWidget</extends>
   <header location="global">markdowneditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
